CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.16.0 (2020-03-04)
+
+- Set the viewport margin top configuration option to `SkyViewkeeperHostOptions` when the omnibar is enabled. [#227](https://github.com/blackbaud/skyux-sdk-builder/pull/227)
```

Comment:
Might not matter, but "viewport margin top configuration option" seems kinda dense. Would it change the meaning if we changed the beginning to "Set the configuration option for the viewport's top margin to `SkyViewKeeperHostOptions`..."? I think that's more readable and easier to understand at first glance, but I'm not sure if changing "viewport margin top" to "viewport's top margin" changes the meaning. If so, we could change the start to "Set the configuration option for the viewport margin top to..." instead. ... Or if you think neither change is necessary, let me know and I'll approve the PR as is.